src/ctia/http/routes/bulk.clj

```diff
@@ -57,27 +68,47 @@
                    :sighting       read-sighting
                    :ttp            read-ttp) %))
 
+(defn with-long-id-fn
+  "return the with-long-id function provided an entity type key"
+  [k]
+  (case k
```

Comment:
Given that it's a value to value mapping (keyword to function), then wouldn't a map literal be better?